### ISSUE TITLE
window.intlTelInput is not a function

### ISSUE THREAD

**Issue by Norrch2** [jackocnr/intl-tel-input#1207]
Hello,

I am following the instructions on getting started without a bundler and I am unable to get the example to work locally. 

When I try to view the page I am getting a console error 'Uncaught TypeError: window.intlTelInput is not a function'. I am using Chrome version 89.0.4389.72.

Here is my index.html
```
<!doctype html>
<html lang="en">
 <head>
  <meta charset="UTF-8">
  <link rel="stylesheet" href="css\intlTelInput.scss"/>
  <title>Sample</title>
 </head>
 <body>
    <div>
        <input type="tel" id="phone"></input>
    </div>
    
	<script src="js\intlTelInput.js"></script>
    <script>
            var input = document.querySelector("#phone");
            
            window.intlTelInput(input, {
            // any initialisation options go here
            });
      </script>
 </body>
</html> 
```

I get a screen with a plain input box and the following in the google console
![image](https://user-images.githubusercontent.com/16027559/110473654-c606d980-80ac-11eb-8fd8-c6721091a352.png)

### Steps to reproduce
1.  Follow Instructions on [Getting Started Not Using a Bundler](https://github.com/jackocnr/intl-tel-input#getting-started-not-using-a-bundler)
2.  View the page in a browser
3.  Open the Console

### Expected behaviour
The window to display a basic telephone number input field with INTL-Tel-Input support

### Actual behaviour
I get an error "window.intlTelInput is not a function'

### Initialisation options
no initialisation options.

**Comment by alouarrani** [comment]
Hello,

Angular 9, node 14, chrome 89.0.4389.90 and intl-tel-input 12.0.12 here.
I've got the same issue with just one difference : 
![image](https://user-images.githubusercontent.com/65347152/111813665-c6d41280-88d9-11eb-9858-896fc6e0951d.png)

**Comment by mattkurek** [comment]
I'm having the same issue. Tried the example, tried using jquery instead of querySelector, tried getElementById. Couldn't get it to work.

**Comment by djamesfar** [comment]
Using laravel jetstream with livewire, same issue.
Looking in intlTelInput.js, indeed there is no function or object of any kind with the name 'intlTelInput'.  
version 17.0.12

**Comment by djamesfar** [comment]
I had to revert to 17.0.8 to get this to work.

**Comment by mtarregh** [comment]
Having the same issue with 17.0.12 with Angular 11 and node 14.16.0

**Comment by sandeep537** [comment]
> Hello,
> 
> Angular 9, node 14, chrome 89.0.4389.90 and intl-tel-input 12.0.12 here.
> I've got the same issue with just one difference :
> ![image](https://user-images.githubusercontent.com/65347152/111813665-c6d41280-88d9-11eb-9858-896fc6e0951d.png)

Facing same issue. Did anyone got the sol

**Comment by toplisek** [comment]
I have tested and tried to move to lower version but did not manage.
Uncaught TypeError: window.intlTelInput is not a function
How to solve this?

**Comment by pbrunisholz** [comment]
Hello, 

If I'm not wrong I think you're not using the right files.

I see that the file you're importing for css is :
`intlTelInput.scss`

 which comes from /src directory when I think you should use files from /build directory.
 
 I guess you did the same for js files. 

What do you say ?

**Comment by Mihard2** [comment]
> Мне пришлось вернуться к 17.0.8, чтобы это работало.
I agree. Seconded. Version 17.0.12 is not working. Except for an error in the console - it is not able to cause anything.

**Comment by Mihard2** [comment]
> I had to revert to 17.0.8 to get this to work.

I agree. Seconded. Version 17.0.12 is not working. Except for an error in the console - it is not able to cause anything.

**Comment by jackocnr** [comment]
Yes I think pbrunisholz might be on to something - please can you guys (@Norrch2 etc) please confirm you are using the files from the build folder, rather than the src folder?

If you've checked and you're still seeing the problem with the build files, then please can you find the minimal set of code that reproduces the problem and put that in a CodePen, then I'd be happy to take a look. Here is [an example pen](https://codepen.io/jackocnr/pen/yehvE) with the plugin up and running - feel free to fork this if it helps.

**Comment by Leonsito** [comment]
Hi, I'm facing the same issue with 17.0.13 and using the files from the build folder. This is the line (145) that throws the error:

`this.hadInitialPlaceholder = Boolean(input.getAttribute("placeholder"));`

> Uncaught TypeError: input.getAttribute is not a function

Btw, I'm using the library in an ASP.NET MVC solution.

**Comment by toplisek** [comment]
I have checked your code again and used CodePen.
It is an error in the line: ))((function(e) {
jQuery("img").error((function() {
    jQuery(this).hide()
}
))((function(e) {
    "object" == typeof module && module.exports ? module.exports = e() : window.intlTelInput = e()
}
))((function(e) {

**Comment by yogesh7076** [comment]
> 'intlTelInput'.
> version 17.0.12

how to solve the problem

**Comment by dmitry-pavlov** [comment]
Getting `window.intlTelInput is not a function` error in Blazor WASM 

CSS and JS files are linked in HEAD as desribed in this post [How to build international phone number input in HTML and JavaScript](https://www.twilio.com/en-us/blog/international-phone-number-input-html-javascript)
```
    <link href="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.8/css/intlTelInput.css" rel="stylesheet" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.8/js/utils.js"></script>

```

```
Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: window.intlTelInput is not a function
      TypeError: window.intlTelInput is not a function
```

**Comment by leonardomv1981** [comment]
I have this problem too. My mistake was that i didnt include: <script src="https://cdn.jsdelivr.net/npm/intl-tel-input@23.1.0/build/js/intlTelInput.min.js"></script>
Works fine for now

**Comment by Sothirich** [comment]
Here is the solution:
  <script src="https://cdn.jsdelivr.net/npm/intl-tel-input@23.8.1/build/js/intlTelInput.min.js"></script>
  <script type="module" src="https://cdn.jsdelivr.net/npm/intl-tel-input@23.8.1/build/js/utils.js"></script>

You need to add script build/js/utils.js too

**Comment by toplisek** [comment]
Is it valid to put <script type="module" src="https://cdn.jsdelivr.net/npm/intl-tel-input@23.8.1/build/js/utils.js"></script> outside function ( //utilsScript: "../intlTelInput/js/utils.js",-)
Can be improved bottom code due to updated version or there is updated demo?
`
var intlTel = (function ()
 {
  function init() {
   function getIp(callback) {
    fetch('https://ipinfo.io/json?token=XXX', {headers: {'Accept': 'application/json'}})
    .then((resp) => resp.json())
    .catch(() =>
     {
      return {
      country: 'de',
      };
     }
    )
    .then((resp) => callback(resp.country));
   }

   var input = document.querySelector("#val_num_phone1");
   window.intlTelInput(input,
    {
     // allowDropdown: false,
     // autoHideDialCode: false,
     // autoPlaceholder: "off",
     // dropdownContainer: document.body,
     // excludeCountries: ["us"],
     // formatOnDisplay: false,
     // geoIpLookup: function(callback) {
     //   $.get("http://ipinfo.io", function() {}, "jsonp").always(function(resp) {
     //     var countryCode = (resp && resp.country) ? resp.country : "";
     //     callback(countryCode);
     //   });
     // },
     // hiddenInput: "full_number",
    initialCountry: "auto",
     // localizedCountries: { 'de': 'Deutschland' },
     // nationalMode: false,
     // onlyCountries: ['us', 'gb', 'ch', 'ca', 'do'],
     // placeholderNumberType: "MOBILE",
    preferredCountries:['fr', 'de', 'at', 'it', 'hr', 'rs'],
     // separateDialCode: true,
    geoIpLookup: getIp,
    //utilsScript: "../intlTelInput/js/utils.js",
    }
   );
  }
  return {
   init
  };
 }()
);
`
